spec/customLibraryCardMenu.component.spec.js

```diff
@@ -0,0 +1,65 @@
+const customLibraryCardMenuItems = __fixtures__['customLibraryCardMenuItems'];
+
+describe('customLibraryCardMenu component', () => {
+  beforeEach(module('customLibraryCardMenu', ($provide) => {
+    $provide.constant("customLibraryCardMenuItems", customLibraryCardMenuItems);
+    $provide.value("translateFilter", (original) => `{${original}}`);
+  }));
+
+  let $compile, $rootScope, element;
+
+  beforeEach(inject(function(_$compile_, _$rootScope_){
+    $compile = _$compile_;
+    $rootScope = _$rootScope_;
+
+    const scope = $rootScope.$new();
+    element = $compile("<prm-library-card-menu-after />")(scope);
+    scope.$digest();
+  }));
+
+  describe('template layout', () => {
+
+    it('should render proper number of buttons', () => {
+      const buttons = element.children();
+
+      Array.from(buttons).forEach(btn => {
```

Comment:
@etgrieco is this how you got away from the `for` loops?

Reply:
yes, but I also had to pollyfill Array.from to ensure it worked in Karma. One more dependency for cleaner code.